pkg/features/kube_features.go

```diff
@@ -468,6 +468,12 @@ const (
 	//
 	// Schedule pods evenly across available topology domains.
 	EvenPodsSpread featuregate.Feature = "EvenPodsSpread"
+
+	// owner: @m1093782566
+	// alpha: v1.16
+	//
+	// Enables service topology aware service routing
+	TopologyAwareServiceRouting featuregate.Feature = "TopologyAwareServiceRouting"
```

Comment:
Please move to a standalone - first - commit

Reply:
done

Reply:
Thoughts on just `ServiceTopology`?